client/src/app/components/manage-organizations/organization-view/edit-organization/edit-organization.component.ts

```diff
@@ -34,7 +34,14 @@ export class EditOrganizationComponent extends OrganizationViewComponent {
         this.isActive = organization.status ? this.activeStates[0] : this.activeStates[1];
         this.originalStatus = organization.status;
         this.orgName = organization.org_name;
-        this.location = organization.address;
+        let locationAddress = '';
+        for (let i = 0; i < organization.address.length; i++) {
+          // tslint:disable-next-line:max-line-length
+          if (organization.address[i] !== '[' && organization.address[i] !== ']' && organization.address[i] !== ' ' && organization.address[i] !== '\'') {
+              locationAddress += organization.address[i];
```

Comment:
You could reduce this using contains if you want

Reply:
Fixed, using Regex to replace the string.